TSPL.docc/LanguageGuide/AccessControl.md

```diff
@@ -44,17 +45,36 @@ A *source file* is a single Swift source code file within a module
 Although it's common to define individual types in separate source files,
 a single source file can contain definitions for multiple types, functions, and so on.
 
+A *package* is a group of modules that you develop as a unit.
+You define the modules that form a package
+as part of configuring the build system you're using,
+not as part of your Swift source code.
+For example, if you use Swift Package Manager to build your code,
+you define a package in your `Package.swift` file
+using APIs from the [PackageDescription][] module;
+if you use Xcode, you specify the package name
```

Comment:
```suggestion
using APIs from the [PackageDescription][] module.
If you use Xcode, you specify the package name
```

Reply:
I preferred the semicolon here, to keep both SwiftPM and Xcode as part of the examples — there are other ways to build Swift code too, like calling `swiftc` directly.  Do you think this would read better with a conjunction?

> For example, if you use SwiftPM ... module, and if you use Xcode ... build setting.

Reply:
Yes, I like that better! I'll update my suggestion.

Reply:
Changed in 8781b5d9.